pkg/providers/cloudstack/cloudstack_test.go

```diff
@@ -193,12 +194,15 @@ func newProviderWithKubectl(t *testing.T, datacenterConfig *v1alpha1.CloudStackD
 
 func newProvider(t *testing.T, datacenterConfig *v1alpha1.CloudStackDatacenterConfig, machineConfigs map[string]*v1alpha1.CloudStackMachineConfig, clusterConfig *v1alpha1.Cluster, kubectl ProviderKubectlClient, cmk ProviderCmkClient) *cloudstackProvider {
 	_, writer := test.NewWriter(t)
+	cmks := CmkClientMap{}
+	cmks["Global"] = cmk
```

Comment:
should we use lowercase names here, given that K8s secret names must be lowercase?